namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/variables.tf

```diff
@@ -42,3 +42,7 @@ variable "slack_channel" {
   description = "Team slack channel to use if we need to contact your team"
   default     = "interventions"
 }
+
+variable "number_cache_clusters" {
+  default = "2"
+}
```

Comment:
_not a blocker, just observation_
The default is also 2: https://github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster/blob/4.2/variables.tf#L46-L49

I think it's fine to be explicit, if we want to be.